Source/Peripheral.swift

```diff
@@ -327,9 +327,15 @@ public class Peripheral {
      */
     public func setNotificationAndMonitorUpdatesForCharacteristic(characteristic: Characteristic)
         -> Observable<Characteristic> {
-            return Observable.of(monitorValueUpdateForCharacteristic(characteristic),
-            setNotifyValue(true, forCharacteristic: characteristic).ignoreElements()
-                .subscribeOn(CurrentThreadScheduler.instance)).merge()
+        return Observable.create { observer in
+            let disposable = Observable.of(self.monitorValueUpdateForCharacteristic(characteristic),
+                self.setNotifyValue(true, forCharacteristic: characteristic).ignoreElements()
+                    .subscribeOn(CurrentThreadScheduler.instance)).merge().subscribe(observer)
+            return AnonymousDisposable {
+                disposable.dispose()
+                self.peripheral.setNotifyValue(false, forCharacteristic: characteristic.characteristic)
```

Comment:
> When you enable notifications for characteristic’s value, the peripheral calls the peripheral:didUpdateNotificationStateForCharacteristic:error: method of its delegate object to notify your app when the characteristic’s value changes.

I'm not sure whether it's called when `false` is set. However it seems not very important in this context.